### PR TITLE
[ADD] field service: default warehouse

### PR DESCRIPTION
**Pull request by zulu-odoo**
task-2948598

### CODE REVIEW

File: content/applications/services/fsm/default_warehouse.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+.. seealso::
+   :doc:`../../inventory_and_mrp/inventory`
+
+Prerequisites
+=============
+
+To set up a user default warehouse, configure the following settings first:
+
+ #. Open the **Inventory** app, then go to :menuselection:`Configuration --> Settings`. Activate
+    **Storage Locations** and **Multi-Step Routes**.
```
Comment (chiaraprattico) [inline content/applications/services/fsm/default_warehouse.rst:20]: How about:

In order to set up a user default warehouse, first go to :menuselection:`Inventory --> Configuration --> Settings` , here activate the :guilabel:Storage Locations` and :guilabel:`Multi-Step Routes`.

File: content/applications/services/fsm/default_warehouse.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ #. Have more than one warehouse in your database.
```
Comment (chiaraprattico) [inline content/applications/services/fsm/default_warehouse.rst:21]: A link to how to do it or to check out the inventory documentation can be helpful.
Or else maybe don't mention it at all if we don't know how to do it.

File: content/applications/services/fsm/default_warehouse.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Set up a default warehouse for a user
+=====================================
```
Comment (chiaraprattico) [inline content/applications/services/fsm/default_warehouse.rst:24]: Here you can continue with the configurations ( or maybe add a new subsection)

File: content/applications/services/fsm/default_warehouse.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+Go to :menuselection:`Settings --> Users & companies --> Manage users`. Then, choose the user from
+the list.
```
Comment (chiaraprattico) [inline content/applications/services/fsm/default_warehouse.rst:27]: Give a little more context here.

File: content/applications/services/fsm/default_warehouse.rst [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
```
Comment (chiaraprattico) [inline content/applications/services/fsm/default_warehouse.rst:28]: erase this and maybe have a single paragraph with the sentence before.

File: content/applications/services/fsm/default_warehouse.rst [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+Go to the :guilabel:`Preferences` tab, then scroll down to **Inventory** catagory. Select the
+default warehouse from the drop-down menu.
+
+Default warehouse in field service tasks
+========================================
+
+Once a default warehouse has been configured for a user, the materials used for a sales order
+linked to a Field Service task will be pulled from that specific warehouse.
```
Comment (chiaraprattico) [inline content/applications/services/fsm/default_warehouse.rst:36]: Make this in present tense if possible.

File: content/applications/services/fsm/default_warehouse.rst (hunk 4 above)
Comment (chiaraprattico) [reply content/applications/services/fsm/default_warehouse.rst:27]: also specify that you are in the :menuselection: `general settings --> manage users`.

File: content/applications/services/fsm/default_warehouse.rst [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+To see that the feature was configured correctly, open the sales order created by the user with
+default warehouse. Go to :guilabel:`Other Info` tab, then scroll down to **Delivery** category. The
+default warehouse is applied correctly on the creation of the sales order.
```
Comment (chiaraprattico) [inline content/applications/services/fsm/default_warehouse.rst:40]: A screenshot can be a nice addition.

**Review by chiaraprattico — CHANGES_REQUESTED** [review]

File: content/applications/services/fsm/default_warehouse.rst (hunk 2 above)
Comment (zulu-odoo) [reply content/applications/services/fsm/default_warehouse.rst:21]: In the end I just explained how to do it & added a link to inventory docs , does that make sense?

File: content/applications/services/fsm/default_warehouse.rst [hunk 8]
```diff
@@ -0,0 +1,51 @@
+===============================
+Set up a user default warehouse
+===============================
+
+Setting up a **default warehouse** can be useful for field technicians who keep a supply in their
+van or those who always resupply in the same warehouse.
+
+With this feature configured, the products in sales orders created during field interventions are
+always pulled from the same warehouse, keeping the inventory accurate.
```
Comment (chiaraprattico) [inline content/applications/services/fsm/default_warehouse.rst:9]: Products in sales orders created during field interventions are always pulled from the same warehouse, keeping the inventory accurate with the :guilabel:` default warehouse1 feature in Odoo.

**Review by chiaraprattico — APPROVED** [review]

File: content/applications/services/fsm/default_warehouse.rst [hunk 9]
```diff
@@ -0,0 +1,45 @@
+===============================
+Set up a user default warehouse
+===============================
+
+Setting up a **default warehouse** can be useful for field technicians who keep a supply in their
+van or those who always resupply in the same warehouse.
```
Comment (jcs-odoo) [inline content/applications/services/fsm/default_warehouse.rst:6]:
```suggestion
van or those who always resupply from the same warehouse.
```

File: content/applications/services/fsm/default_warehouse.rst [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+Products in sales orders created during field interventions are always pulled from the same
+warehouse, keeping the inventory accurate with the :guilabel:`default warehouse` feature in Odoo.
```
Comment (jcs-odoo) [inline content/applications/services/fsm/default_warehouse.rst:9]:
```suggestion
warehouse, keeping the inventory accurate with the default warehouse feature.
```

File: content/applications/services/fsm/default_warehouse.rst [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+
+.. seealso::
+   :doc:`../../inventory_and_mrp/inventory`
+
+Configuration
+=============
+
+In order to be able to set up a user default warehouse :guilabel:`Storage locations` and
+:guilabel:`Multi-step routes` need to be activated. It is also necessary to have more than one
+warehouse in your database.
```
Comment (jcs-odoo) [inline content/applications/services/fsm/default_warehouse.rst:19]:
```suggestion
To set up a user default warehouse, :doc:`storage locations <link>` and
:doc:`multi-step routes <link>` need to be activated. It is also necessary to have more than one
warehouse in your database.
```

File: content/applications/services/fsm/default_warehouse.rst [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+   - :doc:`../../inventory_and_mrp/inventory/management/warehouses/warehouses_locations`
+   - :doc:`../../inventory_and_mrp/inventory/routes/concepts/use_routes`
+
+User account
+------------
+
+To set up a default warehouse for a specific user go to :menuselection:`Settings --> Users -->
+Manage users`. Choose the user you wish to configure a default warehouse for from the list.
+Then, go to the :guilabel:`Preferences` tab, then scroll down to :guilabel:`Inventory`. Select the
+default warehouse from the drop-down menu.
```
Comment (jcs-odoo) [inline content/applications/services/fsm/default_warehouse.rst:31]:
```suggestion
To set up a default warehouse for a specific user, go to :menuselection:`Settings --> Users -->
Manage users`, open a user,
then go to the :guilabel:`Preferences` tab, scroll down to :guilabel:`Inventory`, and select the
default warehouse from the drop-down menu.
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hi @zulu-odoo and @chiaraprattico :)

Thanks for the PR!
Zuzanna, I'll call you to make the last changes together.

**Comment by jcs-odoo** [comment]
@chiaraprattico we also rewrote the last two paragraphs

**Review by jcs-odoo — APPROVED** [review]
Good to go :)

@robodoo delegate+

**Pull request closed without merging**